velox/exec/OutputBufferManager.h

```diff
@@ -94,12 +94,10 @@ class OutputBufferManager {
 
   void removeTask(const std::string& taskId);
 
-  /// Initializes singleton with 'options'. May be called once before
-  /// getInstance().
-  static void initialize(const Options& options);
```

Comment:
@yingsu00 we need to keep this for Prestissimo compatibility. We can remove this once we advance Velox in Prestissimo.
Can you add 
```
#ifdef VELOX_ENABLE_BACKWARD_COMPATIBILITY
   static void initialize(const Options& options) {}
#endif
```